_shows/20_21/freshers_fringe.md

```diff
@@ -0,0 +1,167 @@
+---
+# Check out the Show Docs for more information
+# -- https://history.newtheatre.org.uk/docs/show/
+# Remove or comment any lines that are not being used
+
+title: Freshers Fringe
+playwright: various
+period: Autumn
+season: Online
+season_sort: 360
+date_start: 2020-10-28
+date_end: 2020-10-28
+venue: New Theatre
+
+cast:
+- role: (Caetano and Ella's Group)
```

Comment:
For each of these, it should be of the format:
```
- name: Roni Altman
  note: Caetano and Ella's group
```

Reply:
okay I shall change now